src/sage/interfaces/gap.py

```diff
@@ -1753,7 +1753,7 @@ def intmod_gap_to_sage(x):
         sage: b.parent()
         Ring of integers modulo 65537
     """
-    from sage.rings.finite_rings.all import FiniteField
+    from sage.rings.finite_rings.finite_field_constructor import FiniteField
```

Comment:
I suppose this should be `finite_field_base`?

Reply:
No, line 1762 uses the constructor function, not the class